.github/workflows/ci.yml

```diff
@@ -127,6 +127,12 @@ jobs:
                     sed -i "/mpi4py/ d" requirements.txt
                 fi
                 build_py_project_in_conda_env
+
+                if test "$DOWNSTREAM_PROJECT" = "mirgecom"; then
+                    # Test main branch
+                    examples/run_examples.sh ./examples
+                fi
```

Comment:
Maybe this should be a separate CI job? mirgecom is already the longest job in grudge, I'm reluctant to make CI turnaround even more sluggish.

Reply:
Then maybe drop the downstream `pytest` (which takes a long time)?  Running the examples (typically done within 10 minutes) will be enough and much faster.  The examples won't work if *MIRGE-Com* is broken, and they will also fail if they get anything outside the accepted solution range.

Reply:
[5f5ff6b](https://github.com/inducer/grudge/pull/221/commits/5f5ff6b44eafdfb35ccd0570a60b16f3ecc4f414) implements Mike's suggestion

Reply:
> [5f5ff6b](https://github.com/inducer/grudge/pull/221/commits/5f5ff6b44eafdfb35ccd0570a60b16f3ecc4f414) implements Mike's suggestion

Cool, thanks!  Looks like this cuts the downstream testing expense in half.  Running the examples is more important than running the `pytest` tests.  Having this test in-place would have caught the example failures caused by #222 merge.